Cargo.toml

```
@@ -38,6 +38,10 @@ mimalloc = { version = "0.1.38", default-features = false }
 async-once-cell = {version="0.5.3", default-features=false}
 actix-governor = {version="0.5.0", default-features=false}
 mini-moka = { version="0.10", optional = true, default-features=false, features=["sync"]}
+brotli = { version = "3.4.0", default-features = false, features=["std"], optional=true}
+chacha20poly1305={version="0.10.1", default-features=false, features=["alloc","getrandom"], optional=true}
+base64 = {version= "0.21.5", optional = true}
+chacha20 = {version = "0.9.1", optional = true}
```

```suggestion
chacha20 = {version="0.9.1", default-features=false, optional=true}
base64 = {version="0.21.5", default-features=false, features=["std"], optional=true}
```

I would suggest disabling the unneeded default-features as by disabling them it improves compilation times which in turns helps improve user experience. :slightly_smiling_face: